### PR TITLE
Fix formatting and redundant code in task docs

### PR DESCRIPTION
**Pull request by madhuravi**

### CODE REVIEW

File: modules/task/README.md [hunk 1]
```diff
@@ -38,10 +38,8 @@ func newBackend(host service.Host) (task.Backend, error) {
 
 func runActivity(input string) error {
   // do things
-  results := "results"
-  if err := task.Enqueue(updateCache, input, results); err != nil {
-    return err
-  }
+  results := // calculate results here
```
Comment (alsamylkin) [inline modules/task/README.md]: I am pretty sure this^ will fail to compile :)

Comment (madhuravi) [inline modules/task/README.md]: I'm aware. This is a place holder where stuff goes.

Comment (alsamylkin) [inline modules/task/README.md]: I prefer it to be compilable: 
It makes easy to test examples and for us to think hard about APIs

Minor thing anyway.

File: modules/task/README.md [hunk 2]
```diff
@@ -17,18 +17,19 @@ from the backend, they will execute the function.
 package main
 
 import (
-	"go.uber.org/fx/modules/task"
+  "go.uber.org/fx/modules/task"
   "go.uber.org/fx/service"
+	"go.uber.org/fx/ulog"
```
Comment (alsamylkin) [inline modules/task/README.md]: Formatting?:)

**Pull request merged**